compiler/rustc_middle/src/ty/layout.rs

```diff
@@ -2882,6 +2882,13 @@ where
             return;
         }
 
+        if abi == SpecAbi::X86Interrupt {
+            for arg in &mut self.args {
+                arg.make_indirect_byval();
+            }
+            return;
```

Comment:
Doesn't this need to be gated on LLVM 12?

Reply:
I don't think so. This was previously implicitly byval and is now required to be explicitly byval, but as far as I know making it explicit also works on previous versions.

Reply:
Ok. Can you move it to `adjust_for_cabi` though? This function is for rust specific adjustments only.

Reply:
Done!